crates/forge/tests/integration/cheat_block_number.rs

```diff
@@ -131,7 +132,8 @@ fn cheat_block_number_basic() {
         .unwrap()
     );
 
-    let result = run_test_case(&test, ForgeTrackedResource::CairoSteps);
```

Comment:
There should be a wrapper for doing `run_test_case` with a default UI, so we don't have to add it in every test